### PR TITLE
feat: add Object copy, compose and rewrite RPCs

### PR DESCRIPTION
**Pull request by coryan**
Part of the work for #24

### CODE REVIEW

File: tests/test_emulator_object_special.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+import json
+import os
+import unittest
+from werkzeug.test import create_environ
+
+import emulator
+import testbench
+
+
+class TestEmulatorObjectXML(unittest.TestCase):
```
Comment (cojenco) [inline tests/test_emulator_object_special.py:28]:
```suggestion
class TestEmulatorObjectSpecial(unittest.TestCase):
```

**Comment by coryan** [comment]
Done, PTAL.

File: emulator.py [hunk 2]
```diff
@@ -421,6 +422,155 @@ def object_get(bucket_name, object_name):
...
+    if source_objects is None:
+        testbench.error.missing("source component", None)
+    if len(source_objects) > 32:
+        testbench.error.invalid(
+            "The number of source components provided (%d > 32)" % len(source_objects),
+            None,
+        )
+    composed_media = b""
+    for source_object in source_objects:
+        print("%s\n" % source_object)
```
Comment (cojenco) [inline emulator.py:443]: The code refactor here looks much more clear. Thanks!
Is this print statement for testing purposes?

Comment (coryan) [reply emulator.py]: Doh! I was supposed to remove that. Done.

**Review by cojenco — APPROVED** [review]
Great tests! Thank you!

**Pull request merged**